### PR TITLE
Asserting pseudo-element in PluginTest with AdditionalUtils class

### PR DESCRIPTION
**Pull request by Olgla**

### CODE REVIEW

**Comment by judmi** [comment]
Please create a RF trello card for these changes and attach a link to it here :)

**Comment by Olgla** [comment]
https://trello.com/c/7QY6IIMQ/759-tc0900101-manage-jenkins-plugins-installed-plugins-verify-installed-plugins-contain-ant

**Review by tereshenkov29 — APPROVED** [review]

File: src/test/java/school/redrover/PluginsTest.java [hunk 1]
```diff
@@ -1,39 +1,37 @@
...
 import java.util.List;
 
-
 public class PluginsTest extends BaseTest {
-    public void jsClick(WebElement element) throws InterruptedException {
-
-        JavascriptExecutor executor = (JavascriptExecutor) getDriver();
-        executor.executeScript("arguments[0].click();", element);
-    }
+    AdditionalUtils additionalUtils = new AdditionalUtils();
```
Comment (nikabenz) [inline src/test/java/school/redrover/PluginsTest.java:13]: no needs to create the object of the AdditionalUtils class, bcs all methods are static

File: src/test/java/school/redrover/PluginsTest.java [hunk 2]
```diff
@@ -1,39 +1,37 @@
...
                 "//a[@href = '/manage/pluginManager/installed']"));
-        jsClick(installedPlugins);
 
-        List<WebElement> plugins = getDriver().findElements(By.xpath(
-                "//*[@id='main-panel']/section[2]/div/div[3]/a/dl/dd[1]"));
-        for (WebElement element : plugins) {
-            boolean isSelected = element.isSelected();
-            Assert.assertTrue(isSelected);
+        additionalUtils.jsClick(getDriver(), installedPlugins);
+
```
Comment (nikabenz) [inline src/test/java/school/redrover/PluginsTest.java:24]: We can use AdditionalUtils.jsClick()

File: src/test/java/school/redrover/PluginsTest.java (hunk 1 above)
Comment (Olgla) [reply src/test/java/school/redrover/PluginsTest.java:13]: Good point!

File: src/test/java/school/redrover/PluginsTest.java (hunk 2 above)
Comment (Olgla) [reply src/test/java/school/redrover/PluginsTest.java:23]: Thanks for your review!

**Comment by Olgla** [comment]
creating new

**Pull request closed without merging**